crates/modelardb_compression/src/models/macaque_v.rs

```diff
@@ -13,12 +13,17 @@
  * limitations under the License.
  */
 
-//! Implementation of the Gorilla model type which uses the lossless compression method for
+//! Implementation of the MacaqueV model type which extends the lossless compression method for
 //! floating-point values proposed for the time series management system Gorilla in the [Gorilla
-//! paper]. The compression method described in the paper has been extended with support for lossy
-//! compression by replacing values with the previous value if possible within the error bound. As
-//! this compression method compresses the values of a time series segment using XOR and a variable
-//! length binary encoding, aggregates are computed by iterating over all values in the segment.
+//! paper] by adding support for error-bounded lossy compression, and optimizing flag bits
+//! for better compression of real-life sensor data. MacaqueV adds support for lossy compression by
+//! rewriting the current value with the previous one if possible within the error bound, or
+//! rewriting the least significant mantissa bits of the value to zero within the error bound so that Gorilla
+//! uses fewer bits for encoding. MacaqueV optimizes Gorilla's flag bits by swapping the flag bits
+//! 0 and 10. The experiments showed this modification's effectiveness when Gorilla is used alongside
```

Comment:
"The experiments" are not clear from the context, so I suggest just to write "Experiments showed ...". The intuition could also be explained, i.e. *why* this is good when PMC-Mean and Swing have already been used

Reply:
I fixed it and added one more sentence to give the intuition why this modification is effective.